README.md

```diff
@@ -1,10 +1,78 @@
-# pallet-move
-MoveVM pallet for Substrate-based chains
+# Move Pallet
 
+A pallet for substrate based blockchains to enable the usage of smart contracts written in the Move-language.
 
-# Running pallet from the substrate-node
+## Overview
+
+Smart contracts can directly be implemented and executed as Move scripts or modularized in Move modules. Therefor, the pallet supports publishing of Move modules, and the execution of Move scripts to achieve this functionality. In case of larger projects, the pallet provides the publishing of a bundle (multiple Move modules).
+
+For the execution of Move scripts and the publication of Move modules, the Move source code has to be compiled and serialized into bytecode. For this purpose the tool [`smove`](https://github.com/eigerco/smove) is provided. The tool also provides further helpful features for developing and working with the Move language and this pallet.
+
+For more informations and learning more about how to work with this pallet, the following entry points are provided:
+- [Move Script Example](#move-script-example)
+- [Pallet Configuration in a Substrate-Node](#pallet-configuration-in-a-substrate-node)
+- [Quickstart Guide with a Template-Node](#substrate-node-with-move-pallet)
+- [Benchmarking](#benchmarking)
+
+
+## Move Script Example
+
+As a simplified example a car rental situation will be modeled, by implementing a small Move module and a Move script. For the sake of simplicity, assume that developer Bob already implemented a Move module `CarPool`, which offers the following public method:
+
+```move
+module DeveloperBob::CarPool {
+    // ...
+
+    /// Requests the rent of a car on the given day and between given daytime hours.
+    /// Returns `true` in case of success, otherwise `false`.
+    public fun rent_car(account: &signer, day: u16, hr_from: u16, hr_to: u16, token_limit: u128): bool {
+        // ...
+    }
+
+    // ...
+}
+```
+
+This module will be explained in detail in [our tutorial](TODO), if you are interested in. Bob already published this module and stored it in his own memory space on blockchain. Now we can use this published module by executing the following Move script:
```

Comment:
Too much text for our simple readme - those that have no clue what the Move is about will can the book below
```suggestion
```

Reply:
Done